src/test/end-to-end/contents/CheckYourAnswers-content.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
```

Comment:
is that file name is common for all the Check your answers screen

Reply:
This is for the content of Check your answers for C100 Rebuild only